hackathon_site/dashboard/frontend/src/pages/Teams/Teams.module.scss

```diff
@@ -0,0 +1,15 @@
+@import "./../../assets/abstracts/mixins";
+@import "./../../assets/abstracts/variables";
+
+.teamsDashboard {
+    width: 100%;
+    padding: 10px;
+}
+
+.searchBar {
```

Comment:
This style makes more sense to be placed in TeamSearchBar component instead of here, you also won't need an extra div wrapping the search bar in order to have this style anymore.

Reply:
done